Xamarin.Essentials/PhoneDialer/PhoneDialer.android.cs

```diff
@@ -41,7 +41,9 @@ static void PlatformOpen(string number)
                 phoneNumber = PhoneNumberUtils.FormatNumber(number);
 #pragma warning restore CS0618
 
-            phoneNumber = URLEncoder.Encode(phoneNumber, "UTF-8");
+            // if we are an extension then we need to encode
+            if (phoneNumber.Contains(",") || phoneNumber.Contains(";"))
```

Comment:
Hey @jamesmontemagno, just out of curiosity, why don't you use `char` instead of `string`?

Reply:
could, prob better :)